packages/adminrouter/extra/src/nginx.agent.conf

```diff
@@ -58,5 +58,13 @@ http {
             include common/proxy-headers.conf;
             proxy_pass http://metrics/;
         }
+
+        # Group: System
+        # Description: DC/OS version (unauthenticated)
+        location /dcos-metadata/dcos-version.json {
```

Comment:
Since this is the same location as in `nginx.master.conf`, we should make this one shared location in `common/http.conf`.